plugins/schemas/marc/record.tpl

```diff
@@ -33,7 +33,7 @@
 		<tr valign="top">
 			<td>{if $name}{translate key="plugins.schemas.marc.fields.$name.name"}{else}&mdash;{/if}</td>
 			<td>
-				{foreach from=$entry item=i1val key=i1}{foreach from=$i1val item=i2val key=i2}{foreach from=$i2val item=codeval key=code}{foreach from=$codeval item=value}{$value|escape|default:"&mdash;"}<br/>{/foreach}{/foreach}{/foreach}{/foreach}
+				{foreach from=$entry item=i1val key=i1}{foreach from=$i1val item=i2val key=i2}{foreach from=$i2val item=codeval key=code}{foreach from=$codeval item=value}{$value|trim|nl2br|strip_unsafe_html|default:"&mdash;"}<br/>{/foreach}{/foreach}{/foreach}{/foreach}
```

Comment:
I'm not sure about this one -- previously we treated all fields as plaintext, and this change makes them all HTML-capable. I suspect selective treatment is the only way to get this right.